### PR TITLE
DOCS-2954: Update vision.py

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: src/viam/services/vision/vision.py [hunk 1]
```diff
@@ -93,17 +93,16 @@ async def capture_all_from_camera(
...
         ::
 
-            camera_name = "cam1"
+            my_detector = VisionClient.from_robot(machine, "my_detector")
 
-            # Grab the detector you configured on your machine
-            my_detector = VisionClient.from_robot(robot, "my_detector")
-
-            # capture all from the next image from the camera
+            # // Get the captured data for a camera
```
Comment (jckras) [inline src/viam/services/vision/vision.py:98]: do we want the # and // ?

**Review by jckras — APPROVED** [review]
left one comment about a possible typo, LGTM!

File: src/viam/services/vision/vision.py (hunk 1 above)
Comment (npentrel) [reply src/viam/services/vision/vision.py:98]: good catch - that's a typo

Comment (npentrel) [inline src/viam/services/vision/vision.py:98]:
```suggestion
            # Get the captured data for a camera
```

**Pull request merged**